README.md

```diff
@@ -37,7 +37,15 @@ The course is built using a few tools:
 - [mdbook-i18n-helpers](https://github.com/google/mdbook-i18n-helpers)
 - [mdbook-exerciser](mdbook-exerciser/)
 
-First clone the repository:
+First install rust: 
+
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
I don't like to replicate these instructions — there should only be one place on the Internet that needs to know about the flags-of-the-day to use with `curl` :slightly_smiling_face: That is also why we refer to rustup.sh in the [Installation](https://google.github.io/comprehensive-rust/cargo.html#installation) section.